core-java-modules/core-java-jvm/src/main/java/com/baeldung/classloader/PrintClassLoader.java

```diff
@@ -15,7 +15,7 @@ public class PrintClassLoader {
     public void printClassLoaders() throws ClassNotFoundException {
 
         System.out.println("Classloader of this class:" + PrintClassLoader.class.getClassLoader());
-        System.out.println("Classloader of Logging:" + Logging.class.getClassLoader());
+//        System.out.println("Classloader of Logging:" + Logging.class.getClassLoader());
```

Comment:
let's remove commented code from all the files in this PR including java and XML files

Reply:
Done